apps/crn-frontend/src/shared-research/__tests__/ResearchOutput.test.tsx

```diff
@@ -30,34 +37,44 @@ beforeEach(() => {
   });
 });
 
-const renderComponent = async () => {
+const user = {
+  ...createUserResponse({}, 1),
+  algoliaApiKey: 'algolia-mock-key',
+};
+
+const researchOutputRoute = sharedResearch({}).researchOutput({
+  researchOutputId: id,
+});
+
+const mockToast = jest.fn() as jest.MockedFunction<
+  ContextType<typeof ToastContext>
+>;
+
+const renderComponent = async (path: string) => {
   const result = render(
     <RecoilRoot
       initializeState={({ set }) =>
         set(refreshResearchOutputState(id), Math.random())
       }
     >
-      <Auth0Provider user={{}}>
+      <Auth0Provider user={{ ...user }}>
         <WhenReady>
           <Suspense fallback="Loading...">
-            <MemoryRouter
-              initialEntries={[
-                '/prev',
-                sharedResearch({}).researchOutput({ researchOutputId: id }).$,
-              ]}
-              initialIndex={1}
-            >
-              <Switch>
-                <Route path="/prev">Previous Page</Route>
-                <Route
-                  path={
-                    sharedResearch.template +
-                    sharedResearch({}).researchOutput.template
-                  }
-                  component={ResearchOutput}
-                />
-              </Switch>
-            </MemoryRouter>
+            <ToastContext.Provider value={mockToast}>
```

Comment:
If you were getting errors because of Toast Context missing it means there was an exception saving. I would probably log whats mock toast has been called with in the test that complains about it to see what the error is